examples/alpaca/sft_alpaca.py

```diff
@@ -0,0 +1,103 @@
+import json
+import os
+from argparse import ArgumentParser
+from typing import Dict, List
+
+from datasets import load_dataset
+from transformers import pipeline
+
+import trlx
+from trlx.data.default_configs import TRLConfig, default_sft_config
+
+
+def get_positive_score(scores):
+    "Extract value associated with a positive sentiment from pipeline's output"
+    return dict(map(lambda x: tuple(x.values()), scores))["POSITIVE"]
+
+
+def preprocess(instruction: str, input: str, output: str):
+    """Build Alpaca prompt and output from instruction and input/output examples"""
+    if input:
+        prefix = (
+            "Below is an instruction that describes a task, paired with an input that provides further context."
```

Comment:
Nit: Whitespace after the period is missing, for it to be the same as the original one, currently it's "... further context.Write a response ..."